package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "bedrock-express-browser-fixes",
+  "version": "0.0.1-0",
+  "description": "A Bedrock module for browser fixes.",
+  "main": "./lib",
+  "scripts": {
+    "test-node": "cross-env NODE_ENV=test mocha -r esm --preserve-symlinks -t 30000 -A -R ${REPORTER:-spec} --require test/test-mocha.js test/10-api.js",
+    "coverage": "cross-env NODE_ENV=test nyc --reporter=lcov --reporter=text-summary npm run test-node",
+    "coverage-ci": "cross-env NODE_ENV=test nyc --reporter=lcovonly npm run test-node",
+    "coverage-report": "nyc report",
```

Comment:
see comments below about how the test suite should be setup.  For future reference, basing this repo on this template would have been good:  https://github.com/digitalbazaar/bedrock-module-template-http

There are templates for a number of common use cases: https://github.com/digitalbazaar/?q=template&type=&language=&sort=